Demo for session 4

JolienNoels opened the pull request.

JolienNoels committed:
Demo for session 4

mpaulacaldas commented on exercises/04_rmarkdown/template.Rmd:
````diff
@@ -0,0 +1,120 @@
+library(readxl)
+```
+
+```{r global, echo=FALSE, message=FALSE}
````
I would avoid the name `global` because it means something different in Shiny/Stata. `theme` is a good alternative.

mpaulacaldas reviewed (changes requested):
Hi Jolien! Could you implement the changes I made in this review? It's fine if you don't know how to address them all, but already doing as many as you can would be a huge help.

JolienNoels committed:
Update render.R

I cleaned up the code a bit. I tried to add another layer to create both html and docx using for loop and purrr, but one or the other file format would always be corrupted.

JolienNoels committed:
Update template.Rmd

I updated on all your comments. Everything works with html/docx instead of pdf now
